skaffold.yaml

```diff
@@ -112,6 +112,10 @@ build:
     elasticsearch.host: "onestop-dev-es-http"
     elasticsearch.ssl.certSecret: "onestop-dev-es-http-certs-public"
     elasticsearch.password.valueFrom.secretKeyRef.name: "onestop-dev-es-elastic-user"
+    elasticsearch.snapshot.enabled: true
+    elasticsearch.snapshot.repo: "test-data"
+    elasticsearch.snapshot.bucket: "https://cedar-snapshots.s3.amazonaws.com/elastic-test-data"
+    elasticsearch.snapshot.name: "search-daily-2021.01.25-gc-gxsyzqdo4bpaplrekrq"
```

Comment:
Is there a convenient way to figure out the exact name of an available snapshot that exists in the bucket? When I tested this locally, I had to do some extra `curls` in order to find a snapshot name that was valid. I figured out you can use a wildcard matcher for the name eg. `search-daily-2021.02.26-*` might work, but for example it would be more convenient to go to the S3 bucket in the AWS console and get a snapshot name from there.